anomalib/models/components/layers/sspcab.py

```diff
@@ -0,0 +1,78 @@
+"""Self-Supervised Predictive Convolutional Attention (SSPCAB) block for reconstruction-based models."""
```

Comment:
Can you provide a link to the original paper? Perhaps to the official implementation as well?